lua-tree/share/lua/5.1/kong/pdk/private/node.lua

```diff
@@ -1,103 +1,103 @@
-local log = require "kong.cmd.utils.log"
```

Comment:
Addresses `pl.path requires LuaFileSystem`. This is clumsy, we should probably find a better way around this. I'm also not sure if I need to replace the functionality or if removing it is acceptable.

Also: I had to add a lua `debug.stacktrace()` call [here](https://github.com/Kong/goks/pull/36/files#diff-db381a2809ca54e98d6466b8431c351b0df1aab739441481296048507eb741c2R24) since the go test runner truncates it. I didn't find a toggle that allowed me to show the full stacktrace from the go runner. Is it possible? Would have saved me 30mins of debugging.